Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -526,6 +526,8 @@ static IEnumerable<Instruction> CompiledBindingGetGetter(TypeReference tSourceRe
 				il.Emit(Ret);
 			}
 			else {
+				var locs = new Dictionary<TypeReference, VariableDefinition>();
```

Comment:
if have a feeling that storing vardef using the TypeRef as key might cause problems if the same typeref is found more than once in the binding path...

`{Binding foo.bar.baz.qux}` when foo and baz have the same (value)type, hence the same TypeRef...

Reply:
Such uses of those local variables have exclusive lifetime. The use of local variable which begins with the instruction emitted at line 549 always ends with the instruction emitted at line 558, 593, or 595, and the variable is not reused in the range.